programming_guide/mini_tutorial/README.md

```diff
@@ -167,6 +167,37 @@ More on the Object FIFO data movement patterns in [Section 2b](../section-2/sect
 
 7. Modify the code in [exercise_2](./exercise_2/exercise_2.py) such that the data sizes for each worker are uneven, for example: tile_sizes = [8, 24, 16].
 
+## <ins>Runtime Sequence</ins>
+
+The arguments of the IRON runtime sequence describe buffers that will be available on the host side; the body of the sequence contains commands which describe how those buffers are moved into the AIE-array through `ObjectFifos`.
+
+```python
+data_size = 256
+data_ty = np.ndarray[(data_size,), np.dtype[np.int32]]
+
+# Dataflow with ObjectFifos
+of_in = ObjectFifo(data_ty, name="in")
+of_out = ObjectFifo(data_ty, name="out")
+
+rt = Runtime()
+with rt.sequence(tile_ty, tile_ty) as (a_in, c_out):
+    rt.start(my_worker)
+    rt.fill(of_in.prod(), a_in)
+    rt.drain(of_out.cons(), c_out, wait=True)
+```
+
+Up to five buffers are supported in the runtime sequence, where the fifth is typically used for trace support. This is further described in [Section 4b](../section-4/section-4b/README.md) of the programming guide.
+
+Runtime sequence commands are submitted to and executed by a dedicated command processor in order. Commands that are set to `wait` will block the dedicated command processor until a token associated with their completion is generated. The command processor waits for the runtime sequence to complete before returning by interrupting the host processor.
```

Comment:
> The command processor waits for the runtime sequence to complete before returning by interrupting the host processor.

I'm not sure this is entirely true. The command processor will wait if it's running a wait op. The host code will wait if it's executing an `xrt::wait` call. But there is no requirement that either must wait. A host program could fire off multiple runtime sequences, without waiting, and sync by some other means (e.g. polling on a memory location). One example of this is [an xrt runlist](https://github.com/Xilinx/mlir-aie/blob/main/test/npu-xrt/add_one_two_txn/test.cpp#L143-L179). Are the semantics of an "iron" runtime sequence different?

Reply:
Jeff is correct in his assessment. The current wording is a bit to strong on the absolutes of the synchronization. There are more nuances to consider.